ui/web_gui.py

```diff
@@ -52,17 +57,36 @@ async def index():
 
         @app.get("/api/cameras/")
         async def api_cameras():
+            processor_host = f'http://{self.processor_host}:{self.processor_port}'
             return [{
                 'id': 'default',
+                'storage_host': processor_host,
                 'streams': [
-                    {'src': '/static/gstreamer/default/playlist.m3u8', 'type': 'application/x-mpegURL',
+                    {'src': processor_host + '/static/gstreamer/default/playlist.m3u8', 'type': 'application/x-mpegURL',
                      'birdseye': False},
-                    {'src': '/static/gstreamer/default-birdseye/playlist.m3u8', 'type': 'application/x-mpegURL',
+                    {'src': processor_host + '/static/gstreamer/default-birdseye/playlist.m3u8', 'type': 'application/x-mpegURL',
                      'birdseye': True},
                 ],
             }]
-
+        #@app.get("/static/")
+        #async def redirect():
+        #    response = RedirectResponse('127.0.0.1:8000/static/')
+        #    return response
```

Comment:
Please remove code that's commented out

Reply:
oo, this was for a test and is accidentally committed. thanks I'm going to remove it.